### PR TITLE
add text and buttons to home

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: app/client/templates/pages/homepage.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <template name="homepage">
   <div class="container">
-    <h2>Hello Cruel World</h2>
+    <p>Bienvenido al proceso de registro para participar en la XVII Bienal de Fotografía organizada por el Centro de la Imagen.</p>
+    <a href="/registrarse" class="waves-effect waves-light btn">Comenzar el registro</a>
+    <a href="/registro" class="waves-effect waves-light btn">Reanudar tu registro</a>
```
Comment (varffs) [inline app/client/templates/pages/homepage.html:5]: Shouldnt this be conditional to `is logged in` and either take you to login or /registro? Even tho /resistro will redirect that seems less tidy.

**Comment by mixcas** [comment]
Yeah, is not bad idea.

**Comment by mixcas** [comment]
no, wait, but…

I go to to home and if im logged in i get redirected to the application
If im not, nothing?

**Comment by varffs** [comment]
No for the link itself. 2 link options depending on if logged in.

**Comment by mixcas** [comment]
OHH!
sure!

**Comment by varffs** [comment]
ooops didnt mean to do that lol

**Pull request merged**